### PR TITLE
SCMOD-5074: Resolve dependency issues

### PR DESCRIPTION
**Pull request by thilagasanthosh**
https://autjira.microfocus.com/browse/SCMOD-5074

http://sou-jenkins2.hpeswlab.net/job/JobService/job/JobService~job-service~SCMOD-5074~CI/

### CODE REVIEW

**Review by michael-bryson — CHANGES_REQUESTED** [review]
Found the following unused dependencies in the developer build log:
```
[WARNING] Unused declared dependencies found:
[WARNING]    com.fasterxml.jackson.jaxrs:jackson-jaxrs-base:jar:2.9.6:compile
[WARNING]    com.fasterxml.jackson.core:jackson-core:jar:2.9.6:compile
[WARNING]    com.fasterxml.jackson.jaxrs:jackson-jaxrs-json-provider:jar:2.9.6:compile
```

File: pom.xml [hunk 1]
```diff
@@ -59,20 +216,25 @@
...
             <dependency>
-                <groupId>com.github.workerframework</groupId>
-                <artifactId>worker-tracking-report</artifactId>
-                <version>${caf.worker-framework.version}</version>
+                <groupId>org.powermock</groupId>
+                <artifactId>powermock-module-junit4</artifactId>
+                <version>1.6.4</version>
             </dependency>
             <dependency>
                 <groupId>com.github.workerframework</groupId>
```
Comment (michael-bryson) [inline pom.xml]: Should this be at the top of the list as it is an import dependency?

File: job-service/pom.xml [hunk 2]
```diff
@@ -16,237 +16,264 @@
...
+            <groupId>io.swagger</groupId>
+            <artifactId>swagger-jersey-jaxrs</artifactId>
+            <scope>runtime</scope>
+        </dependency>
+        <dependency>
+            <groupId>com.sun.jersey</groupId>
+            <artifactId>jersey-core</artifactId>
+            <scope>runtime</scope>
+        </dependency>
+        <!--    <dependency>
```
Comment (michael-bryson) [inline job-service/pom.xml]: Why is this commented out, if it is not needed it should be removed?

File: job-service/pom.xml [hunk 3]
```diff
@@ -16,237 +16,264 @@
...
+          <groupId>com.sun.jersey</groupId>
+          <artifactId>jersey-json</artifactId>
+          <version>${jersey-version}</version>
+        </dependency>-->
+        <dependency>
+            <groupId>com.sun.jersey</groupId>
+            <artifactId>jersey-servlet</artifactId>
+            <scope>runtime</scope>
+        </dependency>
+        <!--    <dependency>
```
Comment (michael-bryson) [inline job-service/pom.xml]: Why is this commented out, if it is not needed it should be removed?

File: job-service/pom.xml [hunk 4]
```diff
@@ -16,237 +16,264 @@
...
+        <dependency>
+            <groupId>commons-codec</groupId>
+            <artifactId>commons-codec</artifactId>
+        </dependency>
+        <!-- RabbitMQ queuing services support -->
+        <dependency>
+            <groupId>com.github.cafapi</groupId>
+            <artifactId>caf-api</artifactId>
+        </dependency>
+        <!--    <dependency>
```
Comment (michael-bryson) [inline job-service/pom.xml]: Why is this commented out, if it is not needed it should be removed?

File: job-service/pom.xml [hunk 5]
```diff
@@ -16,237 +16,264 @@
...
+          <groupId>com.github.workerframework</groupId>
+          <artifactId>worker-configs</artifactId>
+        </dependency>-->
+        <!--    <dependency>
```
Comment (michael-bryson) [inline job-service/pom.xml]: Why is this commented out, if it is not needed it should be removed?

File: job-service/pom.xml [hunk 6]
```diff
@@ -16,237 +16,264 @@
...
+          <groupId>com.github.workerframework</groupId>
+          <artifactId>worker-queue-rabbit</artifactId>
+        </dependency>-->
+        <dependency>
+            <groupId>com.github.workerframework</groupId>
+            <artifactId>util-rabbitmq</artifactId>
+        </dependency>
+        <!-- Support for JSON in Response -->
```
Comment (michael-bryson) [inline job-service/pom.xml]: Why is this commented out, if it is not needed it should be removed?

File: job-service/pom.xml [hunk 7]
```diff
@@ -16,237 +16,264 @@
...
+        <!--    <dependency>
+          <groupId>com.fasterxml.jackson.core</groupId>
+          <artifactId>jackson-core</artifactId>
+        </dependency>-->
+        <!--    <dependency>
```
Comment (michael-bryson) [inline job-service/pom.xml]: Why is this commented out, if it is not needed it should be removed?

File: job-service/pom.xml [hunk 8]
```diff
@@ -16,237 +16,264 @@
...
+                    <artifactId>log4j-api</artifactId>
+                    <groupId>org.apache.logging.log4j</groupId>
+                </exclusion>
+                <exclusion>
+                    <artifactId>slf4j-api</artifactId>
+                    <groupId>org.slf4j</groupId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+        <!--    <dependency>
```
Comment (michael-bryson) [inline job-service/pom.xml]: Why is this commented out, if it is not needed it should be removed?

File: job-service/pom.xml [hunk 9]
```diff
@@ -16,237 +16,264 @@
```
Comment (michael-bryson) [inline job-service/pom.xml]: Why is there so much change in this file, it is showing huge changes. Please clean this up so it makes it possible to review.

**Review by michael-bryson — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (thilagasanthosh) [reply pom.xml]: Fixed

File: job-service/pom.xml (hunk 2 above)
Comment (thilagasanthosh) [reply job-service/pom.xml]: Removed commented out block.

File: job-service/pom.xml (hunk 3 above)
Comment (thilagasanthosh) [reply job-service/pom.xml]: Removed commented out block.

File: job-service/pom.xml (hunk 4 above)
Comment (thilagasanthosh) [reply job-service/pom.xml]: Removed commented out block.

File: job-service/pom.xml (hunk 6 above)
Comment (thilagasanthosh) [reply job-service/pom.xml]: Removed commented out block.

File: job-service/pom.xml (hunk 7 above)
Comment (thilagasanthosh) [reply job-service/pom.xml]: Removed commented out block.

File: job-service/pom.xml (hunk 9 above)
Comment (thilagasanthosh) [reply job-service/pom.xml]: Fixed

**Review by michael-bryson — APPROVED** [review]

**Pull request merged**